pandas/core/frame.py

```diff
@@ -2067,6 +2067,12 @@ def to_html(self, buf=None, columns=None, col_space=None, header=True,
             A css id is included in the opening `<table>` tag if specified.
 
             .. versionadded:: 0.23.0
+
+        render_links : boolean, default False
```

Comment:
i think our standard is for bool, @datapythonista should probably validate this generally as I seem mixed usage all over the place.